packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -118,9 +110,11 @@ export function InserterBlockList( {
 	}, [ filterValue, items, categories, collections ] );
 
 	const childItems = useMemo( () => {
-		return filteredItems.filter( ( { name } ) =>
-			includes( rootChildBlocks, name )
-		);
+		return rootChildBlocks
+			? filteredItems.filter( ( { name } ) =>
+					rootChildBlocks?.includes( name )
+			  )
+			: [];
```

Comment:
I added in an optimization here so that if rootChildBlocks is nullish, the filter will not run and an empty array will be returned. Before, it would have run the filter callback multiple times even though it would always return false.

Reply:
Also, Prettier or ESLint is adding some spaces here that I don't think should be there.